start-android.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
out of curiosity again, do you think doing something similar would work for ios? Just in case support for ios is needed later on in the future?

Reply:
I think yes, but the steps would be quite different. The stuff for android in this PR seemed pretty specific so I can't really say for ios very well.